client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -80,55 +122,58 @@ export default function SitesTableRow( { site }: SiteTableRowProps ) {
 		site.is_coming_soon || ( site.is_private && site.launch_status === 'unlaunched' );
 	const isP2Site = site.options?.is_wpforteams_site;
 
+	const displayStatusBadge = isComingSoon || site.is_private;
+
 	return (
-		<Row>
-			<td>
-				<ListTile
-					leading={
-						<a
-							style={ { display: 'block' } }
-							href={ getDashboardUrl( site.slug ) }
-							title={ __( 'Visit Dashboard' ) }
-						>
-							<SiteIcon siteId={ site.ID } size={ 50 } />
-						</a>
-					}
-					title={
-						<div style={ { display: 'flex', alignItems: 'center', marginBottom: '8px' } }>
-							<SiteName style={ { marginRight: '8px' } }>
-								<a href={ getDashboardUrl( site.slug ) } title={ __( 'Visit Dashboard' ) }>
-									{ site.name ? site.name : __( '(No Site Title)' ) }
-								</a>
-							</SiteName>
-							{ isP2Site && <SitesP2Badge>P2</SitesP2Badge> }
-						</div>
-					}
-					subtitle={
-						<div>
-							<SiteUrl
-								href={ site.URL }
-								target="_blank"
-								rel="noreferrer"
-								title={ site.URL }
-								style={ { marginRight: '8px' } }
-							>
-								{ displaySiteUrl( site.URL ) }
-							</SiteUrl>
-							{ site.is_private && ! isComingSoon && (
-								<SitesLaunchStatusBadge>Private</SitesLaunchStatusBadge>
-							) }
-							{ isComingSoon && <SitesLaunchStatusBadge>Coming soon</SitesLaunchStatusBadge> }
-						</div>
-					}
-				/>
-			</td>
-			<td className="sites-table-row__mobile-hidden">{ site.plan.product_name_short }</td>
-			<td className="sites-table-row__mobile-hidden">July 16, 1969</td>
-			<td style={ { width: '20px' } }>
-				<EllipsisMenu>
-					<VisitDashboardItem site={ site } />
-				</EllipsisMenu>
-			</td>
-		</Row>
+		<ClassNames>
+			{ ( { css } ) => (
```

Comment:
This `<ClassNames>` business really is the worst...

Reply:
I will create a PR adding [`@emotion/css`](https://www.npmjs.com/package/@emotion/css#css) and removing the wrapper component. It is definitely the worst.